FFXIVClientStructs/FFXIV/Client/Game/QuestManager.cs

```diff
@@ -19,6 +19,7 @@ public unsafe partial struct QuestManager
 	[FieldOffset(0xB70)] public fixed byte BeastReputation[0x10 * 17];
 	[FixedSizeArray<LeveWork>(16)]
 	[FieldOffset(0xC80)] public fixed byte LeveQuests[0x18 * 16];
+    [FieldOffset(0xE00)] public byte NumLeveAllowances;
```

Comment:
tab error

Reply:
The whole file, or even repository, is mixed with spaces and tabs, sadly.

Reply:
Ah yeah, at first glance it seemed like that was the only line, but it's actually the whole file haha. Not that it matters too much.